src/app/core/services/phase.service.ts

```diff
@@ -125,9 +125,9 @@ export class PhaseService {
    * Retrieves the repository url from local storage and sets to current repository.
    */
   async initializeCurrentRepository() {
-    const org = window.localStorage.getItem('org');
-    const repoName = window.localStorage.getItem('dataRepo');
-    this.logger.info(`PhaseService: received initial org (${org}) and initial name (${repoName})`);
```

Comment:
Let's not change the logging message in the PR

Reply:
Okay. I have updated it accordingly.